### PR TITLE
Straw to bedpe

### PR DESCRIPTION
**Pull request by cfauci**
# Description
made the hic package, created some reading and comparison functions and made testing for Read function, passes tests
## Relevant Links

- [Display Text](https://www.vertgenlab.org/)

### Checklist before requesting a review

- [x] I performed a self-review of my code
- [x] If it this a core feature, I have added thorough tests
- [x] The command `go fmt` was used on all files included

### Testing

None

### Screenshots & Media

None

### Edge cases / Breaking Changes / Known Issues

None

### CODE REVIEW

File: hic/hic.go [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+	}
+
+	for i = range a {
+		equal = Equal(a[i], b[i])
+		if !equal {
+			return equal
+		}
+	}
+
+	return equal
```
Comment (rimangan) [inline hic/hic.go:81]: It might be more readable to 'return true' here and to 'return false' on line 77.

File: hic/hic.go [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+	return current
+}
+
+// Equal compares two Straw structs to see if the values are identical and returns a bool
+func Equal(a Straw, b Straw) bool {
+	var equal bool
+	if a.Bin1Start == b.Bin1Start && a.Bin2Start == b.Bin2Start && a.ContactScore == b.ContactScore {
+		equal = true
+	} else if a.Bin1Start == b.Bin2Start && a.Bin2Start == b.Bin1Start && a.ContactScore == b.ContactScore { //bins don't have restrictions of ordering
+		equal = true
```
Comment (rimangan) [inline hic/hic.go:57]: instead of equal=true this line could just read 'return true'. No need for the equal variable I think

File: hic/testdata/strawTestFile.straw [hunk 3]
```diff
@@ -0,0 +1,2 @@
+1000	5000	5
```
Comment (rimangan) [inline hic/testdata/strawTestFile.straw:1]: Might be good to switch Bin1Start and Bin2Start for one of these (to test for the lack of restriction on ordering)

File: hic/hic.go (hunk 2 above)
Comment (cfauci) [reply hic/hic.go:57]: good point, I'll update

File: hic/testdata/strawTestFile.straw (hunk 3 above)
Comment (cfauci) [reply hic/testdata/strawTestFile.straw:1]: I changed the values in the data struct instead of the file

**Review by rimangan — APPROVED** [review]

**Review by sweavs111 — APPROVED** [review]
looks great!
